### PR TITLE
Write down MSRV policy

### PR DESCRIPTION
**Pull request by workingjubilee**
This was effectively Alex Crichton's policy, so this reflects only the decision to formalize a policy.

### CODE REVIEW

**Comment by ChrisDenton** [comment]
As this is formalizing the policy, does this need some kind of consensus from the crate maintainers team? I mean, perhaps not an fcp but maybe some LGTMs.

It doesn't really change anything, I guess, except that because backtrace was minimally maintained for quite awhile the actual msrv drifted quite a bit from the informal policy.

**Comment by workingjubilee** [comment]
I mean, I de facto cannot merge it without an approval and I cannot approve my own PR, so!

**Comment by ChrisDenton** [comment]
For sure! I just meant like should this be a more than one approval type thing?

**Review by ChrisDenton — APPROVED** [review]

**Comment by workingjubilee** [comment]
Probably!

**Comment by workingjubilee** [comment]
@workingjubilee pseudo-fcp merge

**Comment by workingjubilee** [comment]
Team member @workingjubilee has proposed to merge this. The next step is review by the rest of the tagged team members:

- [ ] @Byron 
- [ ] @joshtriplett 
- [ ] @Amanieu 
- [ ] @thomcc
- [ ] @the8472
- [ ] @ChrisDenton 
- [ ] @KodrAus 
- [ ] @JohnTitor 
- [x] @workingjubilee 

Once a number of reviewers that seem to be "enough" approve (with maybe a few approvals outstanding), this will... probably be just merged. If you spot a major issue that hasn't been raised at any point in this process, please speak up!

cc @rust-lang/crate-maintainers: not-actually-an-FCP proposed for crate-maintainers, please feel free to register concerns.
See [this document](https://github.com/rust-lang/rfcbot-rs/blob/master/README.md) for info about what commands tagged team members can give me while I am pretending to be a substitute rfcbot.

File: README.md
Comment (the8472) [inline README.md:70]: I'm having difficulty parsing this paragraph and how it relates to the previous one.

Comment (workingjubilee) [reply README.md:70]: It's just a statement that this policy as a rule takes precedence over whatever version is tested in CI or similar, since some people get confused about that and the version in CI might lag a bit. I can try to clear up the wording.

Comment (workingjubilee) [reply README.md]: There, I think that's clearer?

**Pull request merged**
